doc/source/performance/procfs-plugin.rst

```diff
@@ -0,0 +1,83 @@
+.. procfs-plugin:
+
+======================================================================
+Using the ProcFS plugin
+======================================================================
+
+
+Implemented in |Percona Server| 8.0.25-15, the ProcFS plugin provides access to the Linux performance counters by running SQL queries against a Percona Server for MySQL 8.0 (or other MySQL 8.0-compatible variant). This plugin is useful for Cloud and MySQL-as-a-service installations.
+
+The plugin reads files from /proc and /sys file systems and returns the file name and any content in rows in the `INFORMATION_SCHEMA.PROCFS` view. The schema definition of this view is:
+
+.. sourcecode:: mysql
+
+    CREATE TEMPORARY TABLE `PROCFS` (
+    `FILE` varchar(1024) NOT NULL DEFAULT '',
+    `CONTENTS` longtext NOT NULL
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+ 
+
+Access to /proc and /sys files and directories is enabled through the system variable `procfs_files_spec`. This variable is global and read only and is set by using either the `mysqld` command line or by editing the :file:`my.cnf`. 
```

Comment:
did we stop using the :variable:`procfs_files_spec` syntax to automatically link to variable description.

Variable description for `procfs_files_spec` is also missing.

Reply:
Yes. We are dropping the :variable: syntax because documentation is dropping psdom.py.